docs/reference/docs/index_.asciidoc

```diff
@@ -31,6 +33,7 @@ The result of the above index operation is:
     "created" : true
 }
 --------------------------------------------------
+// TESTRESPONSE[s/"successful" : 2/"successful" : 1/]
```

Comment:
I switched the example from 10 to 2 because that is the default number of replicas. Then I have to downgrade it to 1 in the test because we only run the test on a single node. We could run the test with two nodes but even then this wouldn't pass (ever? consistently?) because the index is created by the index command and the replicas are replicated from the primary after this returns. At least that is what it looks like to me.